src/types/index.ts

```diff
@@ -974,3 +974,12 @@ export interface LoadApiProviderContext {
   basePath?: string;
   env?: EnvOverrides;
 }
+
+export {
```

Comment:
just a note for @will-holley - Gru tends to over-export things. If you land this or future Gru PRs, remove this block.